plugins/GSdx/GSSettingsDlg.cpp

```diff
@@ -210,25 +217,12 @@ bool GSSettingsDlg::OnCommand(HWND hWnd, UINT id, UINT code)
 		case IDC_NATIVERES:
 		case IDC_SHADEBOOST:
 		case IDC_FILTER:
-			if (code == BN_CLICKED)
-				UpdateControls();
-			break;
-		case IDC_ANISOTROPIC:
-			if (code == BN_CLICKED)
-				UpdateControls();
-			break;
+		case IDC_PALTEX:
 		case IDC_HACKS_ENABLED:
 			if (code == BN_CLICKED)
 				UpdateControls();
 			break;
-		case IDC_SHADEBUTTON:
-			if (code == BN_CLICKED)
-				ShadeBoostDlg.DoModal();
-			break;
-		case IDC_HACKSBUTTON:
-			if (code == BN_CLICKED)
-				HacksDlg.DoModal();
-			break;
+		
```

Comment:
Trailing whitespace. It also wasn't necessary to move IDC_SHADEBUTTON and IDC_HACKSBUTTON.

Please check that you haven't added any whitespace issues after you've reworked your commits.